defect-tracker-data-common/src/main/java/com/defect/tracker/data/dto/ProjectEmp_ResponseDto.java

```diff
@@ -0,0 +1,59 @@
+package com.defect.tracker.data.dto;
+
+public class ProjectEmp_ResponseDto {
+	private long id;
+	private ProjectDto project;
```

Comment:
you can directly use String projectProjectName; this will return only project name;